### PR TITLE
feat(LayoutAnimation): Adds delete animations for layout

### LINKED ISSUE #508: Add remove layout animations.

### PR DESCRIPTION
**Pull request by rozele**
In cases where you want to fade out views or otherwise animate the layout operation that removes a view from the hierarchy, this change makes that possible.

Fixes #508

### CODE REVIEW

File: ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutAnimationController.cs [hunk 1]
```diff
@@ -97,14 +105,43 @@ public void ApplyLayoutUpdate(FrameworkElement view, Dimensions dimensions)
...
+                    X = Canvas.GetLeft(view),
+                    Y = Canvas.GetTop(view),
+                    Width = view.Width,
+                    Height = view.Height,
+                });
+
+            if (animation != null)
+            {
+                view.IsHitTestVisible = false;
+                animation.Finally(@finally);
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutAnimationController.cs]: That's not correct.

File: ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutCreateAnimation.cs [hunk 2]
```diff
@@ -11,6 +11,18 @@ namespace ReactNative.UIManager.LayoutAnimation
     /// </summary>
     class LayoutCreateAnimation : BaseLayoutAnimation
     {
+
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutCreateAnimation.cs]: delete whitespace.

**Comment by mpodwysocki** [comment]
LGTM

File: ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutAnimationController.cs [hunk 3]
```diff
@@ -55,6 +57,12 @@ public void InitializeFromConfig(JObject config)
                 _layoutUpdateAnimation.InitializeFromConfig(updateData, globalDuration);
                 _shouldAnimateLayout = true;
             }
+            var deleteData = config.Value<JObject>("delete");
```
Comment (rozele) [inline ReactWindows/ReactNative/UIManager/LayoutAnimation/LayoutAnimationController.cs]: missing whitespace

**Pull request merged**